rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/ecc/SW360EccService.java

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright Siemens AG, 2017-2019.
+ * Copyright Bosch Software Innovations GmbH, 2017-2018.
+ * Part of the SW360 Portal Project.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.sw360.rest.resourceserver.ecc;
+
+import static com.google.common.base.Strings.nullToEmpty;
+import static java.lang.Math.min;
+import static org.eclipse.sw360.datahandler.common.CommonUtils.nullToEmptyString;
+
+import java.net.URISyntaxException;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TCompactProtocol;
+import org.apache.thrift.protocol.TProtocol;
+import org.apache.thrift.transport.THttpClient;
+import org.apache.thrift.transport.TTransportException;
+import org.eclipse.sw360.datahandler.common.CommonUtils;
+import org.eclipse.sw360.datahandler.common.SW360Utils;
+import org.eclipse.sw360.datahandler.resourcelists.PaginationResult;
+import org.eclipse.sw360.datahandler.thrift.PaginationData;
+import org.eclipse.sw360.datahandler.thrift.components.ComponentService;
+import org.eclipse.sw360.datahandler.thrift.components.Release;
+import org.eclipse.sw360.datahandler.thrift.users.User;
+import org.eclipse.sw360.rest.resourceserver.core.RestControllerHelper;
+import org.jose4j.json.internal.json_simple.JSONArray;
+import org.jose4j.json.internal.json_simple.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.domain.Pageable;
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.hateoas.Link;
+import org.springframework.hateoas.PagedModel;
+import org.springframework.stereotype.Service;
+
+import com.google.common.collect.Maps;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class SW360EccService {
+    @Value("${sw360.thrift-server-url:http://localhost:8080}")
+    private String thriftServerUrl;
+    
+    @NonNull
+    private final RestControllerHelper restControllerHelper;
+    
+    // ECC release view datatables, index of columns
+    private static final String RELEASE_DT_ROW_ECC_STATUS = "status";
+    private static final String RELEASE_DT_ROW_NAME = "name";
+    private static final String RELEASE_DT_ROW_VERSION = "version";
+    private static final String RELEASE_DT_ROW_GROUP = "group";
+    private static final String RELEASE_DT_ROW_ASSESSOR_CONTACT_PERSON = "assessor_contact_person";
+    private static final String RELEASE_DT_ROW_ASSESSOR_DEPARTMENT = "assessor_dept";
+    private static final String RELEASE_DT_ROW_ASSESSMENT_DATE = "assessment_date";
```

Comment:
No need to create the above variables, instead use struct EccInformation{} of components.thrift